package.json

```
@@ -15,14 +15,14 @@
     "main": "index.js",
     "scripts": {
         "build": "mdn-bob",
-        "fix:css": "yarn lint:css --fix",
+        "fix:css": "npm run lint:css --fix",
```

Maybe it's changed, but I'm pretty sure you need the extra `--` to pass arguments to the tasks inside an NPM call
```suggestion
        "fix:css": "npm run lint:css -- --fix",
```